docs/tools/flow-cli/flix.md

```diff
@@ -1,35 +1,74 @@
 ---
-title: Execute Flow Interaction Templates (FLIX) with the Flow CLI
-sidebar_title: Execute FLIX
-description: How to execute Flow Interaction Templates (FLIX) on Flow from the command line
+title: Flow Interaction Templates (FLIX)
+sidebar_title: Execute, Package FLIX
```

Comment:
Would probably go with "Flow Interaction Templates (FLIX)" for the sidebar title too

Reply:
see screenshot below `Flow Interaction Templates (FLIX)` is the side bar title. Any longer and it'll wrap and look crapy. iom